acceptance_tests/features/sensitive_data.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Sample data in the case is sensitive and must be redacted
+
+  Scenario: A case is loaded and sensitive data can be changed
+    Given sample file "sensitive_data_sample.csv" with sensitive column PHONE_NUMBER is loaded successfully
+    When an UPDATE_SAMPLE_SENSITIVE event is received
+    Then the sensitive data on the case is changed
```

Comment:
Same comment r.e. this being specific to `PHONE_NUMBER`, e.g. could be 
`the PHONE_NUMBER in the sensitive data on the case is updated`